### PR TITLE
add `print_sql` parameter; cleanup

### PR DESCRIPTION
**Pull request by mikez**
The new parameter prints every SQL query performed in a legible way.

Example usage is as follows:

```pycon
>>> things.tags(print_sql=True)
/* Query 1 */

SELECT
    uuid, 'tag' AS type, title, shortcut
FROM
    TMTag
WHERE
    TRUE
ORDER BY "index"

...

```

### CODE REVIEW

File: things/database.py [hunk 1]
```diff
@@ -425,191 +425,6 @@ def was_modified_today(self):
         todays_date = datetime.datetime.now().date()
         return last_modified_date >= todays_date
 
-    # -------- Historical methods (TK: transform) --------
-
-    def get_trashed(self):
```
Comment (AlexanderWillner) [inline things/database.py:430]: However, getting trashed tasks would be of interest #16

Comment (mikez) [reply things/database.py:430]: Agreed 👍. Want to give it a try? I will add some remarks to #16 as for how.

**Comment by mikez** [comment]
@AlexanderWillner It's helpful for me if you write LGTM (Looks good to me). Then I know if you've finished reviewing or not. :)

**Comment by AlexanderWillner** [comment]
Not at my laptop yet ;)

**Comment by mikez** [comment]
@AlexanderWillner No rush. :)

**Comment by AlexanderWillner** [comment]
lgtm (looks good to me)

**Pull request merged**